FAQ.md

```diff
@@ -0,0 +1,12 @@
+## Why balenaSense?
+We wanted to simplify the method of building a DIY environmental monitoring application. With balenaSense, users can set up the application, assemble their hardware, and have their own remote-accessible dashboard to view their private environmental data.
+
+## Is balenaSense free?
+Yes! balenaSense requires you to sign up for a free [balenaCloud](https://dashboard.balena-cloud.com/signup) account. Your first 10 devices are always free and full-featured.
+
+## Privacy: is balenaSense secure? 
+balenaSense does not not depend on any type of cloud service to process or store environmental data. Your device will be running balenaOS which does communicate with balenaCloud to get application updates. This can be disabled however and won't affect your experience. Both [balenaSense](https://github.com/balenalabs/balena-sense/) and [balenaOS](https://github.com/balena-os) are open source projects, so feel free to check the source code if you want to know more about how they work!
+
+## Does balenaSense need internet access?
+You don't need an internet connection to use remotely connect to your balenaSense dashboard and view the collected data. Without an internet connection, you won't be able to connect to balenaSense to get host OS or application updates.
```

Comment:
you do need an internet connection

Reply:
I'll reword it as it's a mouthful and awkward as-is.

You need a connection to get it set up and download containers, etc., but the project should work "offline". Am I missing something here? cc @iamsolankiamit @tmigone